modules/ModulesManager/Models/ModuleManager.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Modules\ModulesManager\Models;
+
+use App\Contracts\Model;
+
+/**
+ * Class ModuleManager
+ * @package Modules\ModulesManager\Models
+ */
+class ModuleManager extends Model
+{
+    public $table = 'modules_manager';
+    protected $guarded = [];
+
+    protected $casts = [
+
+    ];
```

Comment:
Missing casts for `enabled` and the timestamps

Reply:
One question. Do timestamps require casts?

Reply:
Yeah nevermind, timestamps don't. I think going forward in 8.x there is a special cast for them, though.